src/common/structure/yaml_writer.go

```diff
@@ -79,10 +79,10 @@ func FindTagsLinesYAML(textLines []string, tagsAttributeName string) common.Line
 	indent := ""
 	for i, line := range textLines {
 		if strings.Contains(line, tagsAttributeName+":") {
-			tagsLines.Start = i
+			tagsLines.Start = i + 1
 			indent = common.ExtractIndentationOfLine(line)
-		} else if common.ExtractIndentationOfLine(line) <= indent && !strings.HasPrefix(line, indent+"-") && tagsLines.Start >= 0 {
-			tagsLines.End = i - 1
+		} else if common.ExtractIndentationOfLine(line) <= indent && tagsLines.Start >= 0 || i == len(textLines)-1 {
```

Comment:
@tronxd Can you add parenthesis around what should be evaluated first, `&&` or `||` ?